File: UI/ES6/Class/Car.js
class Car{

    title;
    constructor(){
        //this.title = 'Car';
    }
    drive(){
        return 'I am driving car';
    }

    static display(){
        console.log('I am displaying')
    }
}

const car = new Car();
console.log(car)
console.log(car.drive());
Car.display();

export default Car;

File: Java/spring-parent-project/spring-aop-config/src/main/java/com/demo/Car.java
package com.demo;

public class Car {


    public void drive() {
        System.out.println("driving the car with speed");
        //System.out.println("loggig from car class");
        System.out.println("driving the car ");
    }

    public void drive(int speed) {
        System.out.println("driving the car with speed " + speed);
    }

    public void stop() {
        System.out.println("stopped the car");
    }
}
